CMake/SlicerCPack.cmake

```
@@ -111,6 +111,29 @@ if(Slicer_BUILD_QT_DESIGNER_PLUGINS)
     )
 endif()
 
+# Get Qt root directory
+get_property(_filepath TARGET "Qt5::Core" PROPERTY LOCATION_RELEASE)
+get_filename_component(_dir ${_filepath} PATH)
+set(qt_root_dir "${_dir}/..")
+
+if(Slicer_BUILD_I18N_SUPPORT)
+  foreach(tool lconvert lrelease lupdate)
+    install(PROGRAMS ${qt_root_dir}/bin/${tool}${CMAKE_EXECUTABLE_SUFFIX}
+      DESTINATION ${Slicer_INSTALL_ROOT}/bin COMPONENT Runtime
+      )
```

```suggestion
      )
      slicerStripInstalledLibrary(
        FILES "${Slicer_INSTALL_ROOT}/bin/${tool}"
        COMPONENT Runtime
        )
```

This is similar to what is done in [CMake/SlicerBlockInstallQt.cmake#L184-L193](https://github.com/Slicer/Slicer/blob/201ab3a1d1bb16938f41db63634bc0190cc08d27/CMake/SlicerBlockInstallQt.cmake#L184-L193) for Qt Designer.

For sake of consistency, I suggest to move this (and the update of `rpath`) to `SlicerBlockInstallQt.cmake`